src/screens/Followings/FollowersScreen.js

```diff
@@ -12,14 +13,17 @@ import {Header} from '../../components';
 import {fonts} from '../../utils/fonts';
 
 function FollowersScreen() {
+  const route = useRoute();
+  const {params} = route;
   const navigation = useNavigation();
   const [profileState, dispatchNavbar] = React.useContext(Context).profile;
 
   const isAndroid = Platform.OS === 'android';
 
   const [isLoading, setIsLoading] = React.useState(false);
-  const [dataFollower, setDataFollower] = React.useState([]);
+  const [dataFollower, setDataFollower] = React.useState(params?.params?.initialData || []);
   const [searchText, setSearchText] = React.useState('');
+  const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here include the introduction of `route` and `params` variables, initialization of `dataFollower` state with `params?.params?.initialData || []`, and a new state variable `isFirstTimeOpen`. These changes seem to be part of an effort to improve the initial loading of follower data. The use of optional chaining (`?.`) ensures that no errors are thrown if `params` or `params.params` are undefined.
```javascript
const route = useRoute();
const {params} = route;
const [dataFollower, setDataFollower] = React.useState(params?.params?.initialData || []);
const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->